newIDE/app/src/AssetStore/AssetsHome.js

```diff
@@ -59,17 +60,108 @@ const useStylesForGridListItem = makeStyles(theme =>
   onPackSelection: string => void,
+  onPrivateAssetPackSelection: PrivateAssetPackListingData => void,
 |};
 
 export const AssetsHome = ({
   assetPacks: { starterPacks },
+  privateAssetPacks,
+  assetPackRandomOrdering,
   onPackSelection,
+  onPrivateAssetPackSelection,
 }: Props) => {
   const classesForGridListItem = useStylesForGridListItem();
   const windowWidth = useResponsiveWindowWidth();
   const gdevelopTheme = React.useContext(ThemeContext);
 
+  const starterPacksTiles = starterPacks.map((pack, index) => (
+    <GridListTile
+      classes={classesForGridListItem}
+      key={pack.tag}
+      tabIndex={0}
+      onKeyPress={(event: SyntheticKeyboardEvent<HTMLLIElement>): void => {
+        if (shouldValidate(event)) {
+          onPackSelection(pack.tag);
+        }
+      }}
+      onClick={() => onPackSelection(pack.tag)}
+    >
+      <Paper
+        elevation={2}
+        style={{
+          ...styles.paper,
+          backgroundColor: gdevelopTheme.list.itemsBackgroundColor,
+        }}
+      >
+        <CorsAwareImage
+          key={pack.name}
+          style={styles.previewImage}
+          src={pack.thumbnailUrl}
+          alt={`Preview image of asset pack ${pack.name}`}
+        />
+        <Column>
+          <Line justifyContent="space-between" noMargin>
+            <Text style={styles.packTitle} size="body2">
+              {pack.name}
+            </Text>
+            <Text style={styles.packTitle} color="primary" size="body2">
+              <Trans>{pack.assetsCount} Assets</Trans>
+              {pack.userFriendlyPrice ? ' - ' + pack.userFriendlyPrice : null}
+            </Text>
+          </Line>
+        </Column>
+      </Paper>
+    </GridListTile>
+  ));
+
+  const privateAssetPacksTiles = privateAssetPacks.map(pack => (
+    <GridListTile
+      classes={classesForGridListItem}
+      key={pack.id}
+      tabIndex={0}
+      onKeyPress={(event: SyntheticKeyboardEvent<HTMLLIElement>): void => {
+        if (shouldValidate(event)) {
+          onPrivateAssetPackSelection(pack);
+        }
+      }}
+      onClick={() => onPrivateAssetPackSelection(pack)}
+    >
+      <Paper
+        elevation={2}
+        style={{
+          ...styles.paper,
+          backgroundColor: gdevelopTheme.list.itemsBackgroundColor,
+        }}
+      >
+        <CorsAwareImage
+          key={pack.name}
+          style={styles.previewImage}
+          src={pack.thumbnailUrls[0]}
+          alt={`Preview image of asset pack ${pack.name}`}
+        />
+        <Column>
+          <Line justifyContent="space-between" noMargin>
+            <Text style={styles.packTitle} size="body2">
+              {pack.name}
+            </Text>
+            <Text style={styles.packTitle} color="primary" size="body2">
+              <Trans>{pack.description}</Trans>
+            </Text>
+          </Line>
+        </Column>
+      </Paper>
+    </GridListTile>
+  ));
+
+  const allTiles = starterPacksTiles
```

Comment:
see comment in provider. I'm wondering if we could have just 1 array of assetPacks already randomized, (with a prop to know if it's public or private) and you just have to loop through it, rather than doing the logic here